code/utilities/helpers/LLMHelper.py

```diff
@@ -10,7 +10,7 @@ def __init__(self):
         env_helper: EnvHelper = EnvHelper()
 
         # Configure OpenAI API
-        openai.api_type = "azure"
+        openai.api_type = env_helper.OPENAI_API_TYPE
         openai.api_version = env_helper.AZURE_OPENAI_API_VERSION
         openai.api_base = env_helper.OPENAI_API_BASE
         openai.api_key = env_helper.OPENAI_API_KEY
```

Comment:
@adamdougal should we add the auth_type logic here to set the api_key as key or token?